### PR TITLE
Fix license headers to match LICENSE file.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7

File: bridge.go [hunk 1]
```diff
@@ -1,8 +1,15 @@
-// Copyright (c) 2013, Prometheus Team
-// All rights reserved.
+// Copyright 2013 Prometheus Team
```
Comment (beorn7) [inline bridge.go]: This should be "The Prometheus Authors".

File: bridge_test.go [hunk 2]
```diff
@@ -1,8 +1,15 @@
-// Copyright (c) 2013, Prometheus Team
-// All rights reserved.
+// Copyright 2013 Prometheus Team
```
Comment (beorn7) [inline bridge_test.go]: And again, and everywhere...

File: bridge.go (hunk 1 above)
Comment (juliusv) [inline bridge.go]: Ah. Right. Then this needs to be fixed in the PGW as well, where I stole this header snippet from :)

**Comment by juliusv** [comment]
Fixed throughout.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**